### DISCUSSION TITLE
LilyGo T-Display ESP32-S3

### DISCUSSION THREAD

**Discussion by Tissy** [Bodmer/TFT_eSPI#1982] [New processor/display support or capability request] [answered]
Hi,

I have the newest Lilygo T-Display ESP32-S3 (https://github.com/Xinyuan-LilyGO/T-Display-S3) however I cannot get the TFT display to work?

Is anyone able to confirm which settings are required in `User_Setup_Select.h` for this board please.

Many thanks.

**Comment by Bodmer** [comment, marked as answer]
That is a different repository copy to this one so ask support from Lilygo.

The setup they seem to use is this one, which has been adapted for that display board:

https://github.com/Xinyuan-LilyGO/T-Display-S3/blob/main/lib/TFT_eSPI/User_Setups/Setup24_ST7789.h
